docs/tutorial/02-hello-world.md

```diff
@@ -143,44 +136,50 @@ Open the Account `0x01` tab with the file called
 ```cadence HelloWorld.cdc
 // HelloWorld.cdc
 //
-pub contract HelloWorld {
+access(all)
+contract HelloWorld {
 
     // Declare a public field of type String.
     //
-    // All fields must be initialized in the init() function.
-    pub let greeting: String
+    // All fields must be initialized in the initializer.
+    access(all)
+    let greeting: String
 
-    // The init() function is required if the contract contains any fields.
+    // The initializer is required if the contract contains any fields.
     init() {
         self.greeting = "Hello, World!"
     }
 
     // Public function that returns our friendly greeting!
-    pub fun hello(): String {
+    // Specified as `view` to declare that it does not change any state
+    access(all) view
+    fun hello(): String {
         return self.greeting
     }
 }
 ```
 
-The line `pub contract HelloWorld ` declares a contract that is accessible in all scopes (public).
-It's followed by `pub let greeting: String` which declares a state constant (`let`) of type `String` that is accessible in all scopes(`pub`).
+The line `access(all) contract HelloWorld ` declares a contract
+that is accessible in all scopes (`access(all)`, typically known as public).
```

Comment:
typically -> previously?

Reply:
typically is right here. I'm referring to how this access level is usually called in other languages